docs/includes/net-standard-table.md

```diff
@@ -0,0 +1,19 @@
+| Platform Name              | Alias       |        |        |        |        |        |        |        |       |
```

Comment:
I think we should add some extra instructions on how to read the table here. See https://github.com/dotnet/standard/pull/243#issuecomment-286926245 for some suggestions from @sam-wheat which I think are pretty good.